matchers/be_numerically_matcher.go

```diff
@@ -13,11 +13,25 @@ type BeNumericallyMatcher struct {
 }
 
 func (matcher *BeNumericallyMatcher) FailureMessage(actual interface{}) (message string) {
-	return format.Message(actual, fmt.Sprintf("to be %s", matcher.Comparator), matcher.CompareTo[0])
+	return matcher.FormatFailureMessage(actual, true)
 }
 
 func (matcher *BeNumericallyMatcher) NegatedFailureMessage(actual interface{}) (message string) {
-	return format.Message(actual, fmt.Sprintf("not to be %s", matcher.Comparator), matcher.CompareTo[0])
+	return matcher.FormatFailureMessage(actual, false)
+}
+
+func (matcher *BeNumericallyMatcher) FormatFailureMessage(actual interface{}, positive bool) (message string) {
+	var formatted string
```

Comment:
Following on from the comment below, maybe we could just rename this to `message`, since it's the thing that _will_ be formatted, instead of the thing that _is_ formatted.